testing/test_runner.py

```diff
@@ -1002,6 +1002,17 @@ def test_func():
         assert rep.capstdout == ""
         assert rep.capstderr == ""
 
+    def test_longrepr_type(self, testdir) -> None:
+        reports = testdir.runitem(
+            """
+            import pytest
+            def test_func():
+                pytest.fail(pytrace=False)
+        """
+        )
+        rep = reports[1]
+        assert not isinstance(rep.longrepr, str)
```

Comment:
I think it would be better to check the actual instance instead of checking that it is not a `str`. 👍

Reply:
Of course, you are right. Thanks!